### PR TITLE
Milestone 4

### PR DESCRIPTION
**Pull request by ZJUAlexChen**
This milestone adds the retrieval step for Text Summarization module. In addition, it also adds the result evaluation module.

### CODE REVIEW

File: public/result_evaluation/evaluate_result.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    @author Zhenke Chen
+    @date 2/11/2021
+
+    Apply Rouge-L to evaluate the Text Summarization result
+    Apply DPR Retrieval to evaluate the relavance between the question and text
+'''
+
+
+# import the required packages
+import rouge
```
Comment (ashuk203) [inline public/result_evaluation/evaluate_result.py]: Is the line 11 import needed? I only see you having used `Rouge`. I think having just the line 12 import is fine

File: README.md [hunk 2]
```diff
@@ -110,7 +111,61 @@ def trainModel( embeddedData, dataWithScores ):
...
 ``` -->
 
-## Module 3: Results Selection and Evaluation
+## Module 2: Text Summarization
+### Function 2 retrieval
+* ***Functionality***: Apply the Dense Passage Retrieval (DPR) to evaluate and rank the relevance between the question and text, then select the most relevant ones as the original text
+* ***Input***:
+   * question: the question asked by the user
+   * raw_text: the raw text fetched from the Data Collection module 
+   * paragraph_num: the number of paragraphs user wants to apply for the original text
```
Comment (ashuk203) [inline README.md]: Can you clarify what "number of paragraphs user wants to apply" means in the argument description for `paragraph_num`? Does this mean the number of paragraphs that are retrieved by the function?

**Review by ashuk203 — CHANGES_REQUESTED** [review]
Everything is looking good! Just need a few details addressed that I commented on below.

File: README.md (hunk 2 above)
Comment (ZJUAlexChen) [reply README.md]: The paragraph_num means the number of paragraphs to compose the original text. I have changed the expression. Thank you.

File: public/result_evaluation/evaluate_result.py (hunk 1 above)
Comment (ZJUAlexChen) [reply public/result_evaluation/evaluate_result.py]: Sorry the line 11 is not needed, I have deleted it.

**Pull request closed without merging**